### PR TITLE
Java wordcount swift s3

### PR DESCRIPTION
**Pull request by mojtabaimani**

### CODE REVIEW

File: pages/platform/data-processing/36_HOWTO_use_object_storage_java/guide.en-asia.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        List<Tuple2<String, Integer>> output = counts.collect();
+        for (Tuple2<?,?> tuple : output) {
+            System.out.println(tuple._1() + ": " + tuple._2());
+        }
+        spark.stop();
+    }
+}
+```
+
+To package this java code and create a jar file you need to create a pom.xml file and build with Maven software with command `mvn package`:
```
Comment (impolitepanda) [inline pages/platform/data-processing/36_HOWTO_use_object_storage_java/guide.en-asia.md:87]: you only need to create a pom.xml file if you're using maven. Rephrase to say that to package, you can use maven, gradle, etc. Here is an example of pom.xml file for maven

File: pages/platform/data-processing/32_HOWTO_fill-job-submit-form/guide.en-asia.md [hunk 2]
```diff
@@ -86,6 +86,10 @@ For Apache Spark, for example, you will also have to :
 
 ![Data Processing Engine](images/configure.png){.thumbnail}
 
+> [!warning]
+> The arguments of the application are stored in plane text. It is advised that you store your credentials in configuration files instead of using arguments.
```
Comment (impolitepanda) [inline pages/platform/data-processing/32_HOWTO_fill-job-submit-form/guide.en-asia.md:90]: plain text

Comment (impolitepanda) [reply pages/platform/data-processing/32_HOWTO_fill-job-submit-form/guide.en-asia.md:90]: Also mention that the configuration files should be put in the same container as the job so they can be uploaded at the same time

File: pages/platform/data-processing/35_HOWTO_use_object_storage/guide.en-asia.md [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager).
+- An OVHcloud account
+- A cloud project in your OVHcloud account (see [How to create a cloud project](../../public-cloud/getting_started_with_public_cloud_logging_in_and_creating_a_project){.external} for details).
+- Data Processing activated (see [How to activate the Data Processing service](../activation){.external} for details).
+- A Conda compatible environment file.
+The following is an example with only dependencies required for this How to: [environment.yml](https://github.com/ovh/data-processing-samples/tree/master/python_objectStorage/environment.yml).
+Otherwise, this [How to](../generate-environment) explains how to generate an environment file.
+- The input file 'wordcount.txt' used by the following samples is available [Here](https://github.com/ovh/data-processing-samples/tree/master/python_objectStorage/wordcount.txt).
+You have to upload it in the same Swift container as the one used for Python samples. Please refer to this documentation that explains how to create a Swift container [Getting started with the Swift API](../../public-cloud/getting_started_with_the_swift_api/).
```
[the remark this answers is not in the document]
Comment (mojtabaimani) [reply pages/platform/data-processing/35_HOWTO_use_object_storage/guide.en-asia.md:27]: > Regarding our discussion, is this requirement still necessary?
> It's definitely a prerequisite for the first part of this how to, but no more, no?

This is needed for the first part. right.

File: pages/platform/data-processing/35_HOWTO_use_object_storage/guide.en-asia.md [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+
+## Read data in the current container with Apache Spark using OpenStack Swift API
+
+Find below the code in Python that reads one object wordcount.txt in the current container and prints the number of occurrences per word.
+
+Save it in a file called 'wordcount.py' or download it from this repository: [Data Processing Samples - Object Storage](https://github.com/ovh/data-processing-samples/tree/master/python_objectStorage/wordcount.py).
```
[the remark this answers is not in the document]
Comment (mojtabaimani) [reply pages/platform/data-processing/35_HOWTO_use_object_storage/guide.en-asia.md:33]: No, it was wordcount.txt not wordcount.py

**Pull request merged**